widgetssdk/src/main/java/com/glia/widgets/view/header/AppBarView.kt

```diff
@@ -105,9 +115,18 @@ class AppBarView @JvmOverloads constructor(
 
     fun showXButton() {
         binding.endButton.isGone = true
+        binding.endScreenSharingButton.isGone = true
```

Comment:
can use the declared `endScreenShareButton` one

Reply:
true, I did some refactoring to eliminate additional code, but forgot to remove it here